src/diffusers/image_processor.py

```diff
@@ -165,10 +172,31 @@ def preprocess(
 
     def postprocess(
         self,
-        image,
+        image: torch.FloatTensor,
         output_type: str = "pil",
-    ):
-        if isinstance(image, torch.Tensor) and output_type == "pt":
+        do_normalize: Optional[Union[List[bool], bool]] = None,
+    ):  
+        if not isinstance(image, torch.Tensor):
+            raise ValueError(
+                f"Input for postprocess is in incorrect format: {type(image)}.  we only support pytorch tensor"
+            )
+        if output_type not in ["latent", "pt", "np", "pil"]:
```

Comment:
Note: the `raise ValueError` that happens later at the end of the function will no longer trigger. https://github.com/huggingface/diffusers/blob/7c1bb9ae81c97f6f10cf36784c17a51ebddeea08/src/diffusers/image_processor.py#L208-L209 We could remove it, or we could leave it there anyway so everything still works after we remove the deprecation warning.

Reply:
oh thanks I think we can just remove it ( can add it back if we ever remove the warning)